tools/ClassGen/NodeGen.cpp

```diff
@@ -216,6 +216,13 @@ int main(int argc, char **argv) {
       .addResultFromCtorArg()
       .setDocstring("Performs elementwise pow(Base, Exp).");
 
+  // clang-format off
```

Comment:
what does clang-format want to rearrange?

Reply:
It puts all calls on the same line, e.g. `BB.newNode("Log").addInput("Input").addResultFromCtorArg().setDocstring(`